packages/suite-native/src/App.tsx

```diff
@@ -4,7 +4,7 @@ import { Provider } from 'react-redux';
 import './support/global';
 // import { Sentry, SentryLog } from 'react-native-sentry';
 import ErrorBoundary from '@suite-support/ErrorBoundary';
-import Preloader from './support/TemporaryPreloader';
+// import Preloader from './support/TemporaryPreloader';
```

Comment:
missing preloader?

Reply:
It's not really working right now anyway. We'll sort it out later.